packages/gatsby/src/schema/graphql-engine/entry.ts

```diff
@@ -195,6 +195,9 @@ export class GraphQLEngine {
     }
   }
 
+  /**
+   * @deprecated use findPageByPath exported from page-ssr module instead
+   */
   public findPageByPath(pathName: string): IGatsbyPage | undefined {
```

Comment:
This is kept for backward-compat, we will not use this anymore when deployed but to not break various integrations this is kept for now